package.json

```diff
@@ -5,7 +5,8 @@
   "private": true,
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "dev": "browser-sync start --server --files 'royale-theme'"
```

Comment:
you can probably simplify `--files 'royale-theme'` to `--files .` which means "current directory" 
with all the quotes it could get confusing when making a change.

Is there a way that we only watch `.css`, `.js` and `.html` files? with the "all' files if something else changes (for example cached files) it may restart browser-sync which may not be what we want.

Reply:
Single quotes are there because this is a command line flag.

But I will make the changes to the --files options.